net-misc/x11-ssh-askpass/x11-ssh-askpass-1.2.4.1-r3.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="X11-based passphrase dialog for use with OpenSSH"
+HOMEPAGE="https://github.com/sigmavirus24/x11-ssh-askpass"
+SRC_URI="https://github.com/sigmavirus24/${PN}/archive/${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="HPND"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+
+RDEPEND="virtual/ssh
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXt"
+DEPEND="${RDEPEND}"
+BDEPEND="app-text/rman
+	sys-devel/gcc
+	>=x11-misc/imake-1.0.8-r1"
```

Comment:
```suggestion
RDEPEND="
	virtual/ssh
	x11-libs/libICE
	x11-libs/libSM
	x11-libs/libX11
	x11-libs/libXt"
DEPEND="${RDEPEND}"
BDEPEND="
	app-text/rman
	sys-devel/gcc
	>=x11-misc/imake-1.0.8-r1"
```

Reply:
I'm looking at new ebuilds with my kde proj trained eye, and usually we do put lists of more-than-one dependencies into a neatly aligned block. @SpiderX: what do you think?